packages/toolkit/src/codewhisperer/service/transformByQHandler.ts

```diff
@@ -24,7 +24,7 @@ import { calculateTotalLatency, javapOutputToTelemetryValue } from '../../amazon
 import { MetadataResult } from '../../shared/telemetry/telemetryClient'
 import request from '../../common/request'
 
-/* TODO: once supported in all browsers and past "experimental" mode, use Intl DurationFormat:
+/* Once supported in all browsers and past "experimental" mode, use Intl DurationFormat:
```

Comment:
Removed `TODO` but keeping the content of the comment. Got feedback from a security engineer that apparently having `TODO`s scattered is not good.

Reply:
Did they say why TODOs were bad?

Reply:
Apparently customers looking through the code? Makes it seem incomplete. I disagreed but that's what I was told. I'm not going to remove the comments themselves so that I can still see them.

Reply:
I suggest pushing back on that.

Reply:
I'll do that. I think his suggestion was to instead create tickets for these items but I prefer a TO-DO over a ticket as it's easier for me to track / remember.

Reply:
In case he doesn't get back to me soon enough, here's a ticket to track this: https://sim.amazon.com/issues/EGCP-97